Tests/Common/ProtobufSerializationTests.cs

```diff
@@ -34,117 +31,6 @@ namespace QuantConnect.Tests.Common
     [TestFixture, Parallelizable(ParallelScope.All)]
     public class ProtobufSerializationTests
     {
-        [Test]
-        public void SerializeRoundTripEstimizeRelease()
```

Comment:
Think we should add the Iconic data types round trip serialization as a test of serializing these custom data types 👍